src/internal/account.ts

```diff
@@ -194,8 +191,14 @@ export async function getAccountTokensCount(args: {
     query: graphqlQuery,
     originMethod: "getAccountTokensCount",
   });
+  if (
+    !data.current_token_ownerships_v2_aggregate.aggregate ||
+    data.current_token_ownerships_v2_aggregate.aggregate.count === undefined
+  ) {
+    throw Error("Failed to get the count of account tokens");
```

Comment:
I think we can simply return `data.current_token_ownerships_v2_aggregate.aggregate.` and if there is no token, it would just be 0

Reply:
don't quite understand.  to return `data.current_token_ownerships_v2_aggregate.aggregate` that would return type 
{count: number} | undefined.  We want to return type number here no?

Reply:
oh I am saying return `data.current_token_ownerships_v2_aggregate.aggregate?.number` and then I think it would return `0` or some number...?

Reply:
ah gotcha, I read the typing wrong.  you're right no need to check for count undefined.